kvdb-rocksdb/src/lib.rs

```diff
@@ -703,7 +703,7 @@ impl Drop for Database {
 
 #[cfg(test)]
 mod tests {
-	extern crate tempdir;
+	use tempdir;
```

Comment:
Not required.

Just remove the `use self::tempdir` below and just use `tempdir::`